test/test.ts

```diff
@@ -189,20 +188,8 @@ test('write::uastc', (t) => {
 });
 
 test('platform::web', (t) => {
-	// Emulate browser API.
-	global.TextEncoder = TextEncoder as any;
-	global.TextDecoder = TextDecoder as any;
-	const _from = Buffer.from;
-	Buffer.from = (() => {
-		throw new Error('Should not be called.');
-	}) as any;
-
-	try {
-		const result = write(read(SAMPLE_UASTC));
-		t.true(result instanceof Uint8Array, 'success');
-	} finally {
-		Buffer.from = _from;
-	}
+	const result = write(read(SAMPLE_UASTC));
+	t.true(result instanceof Uint8Array, 'success');
```

Comment:
Not sure if to delete this test since it not make much sense now.

Reply:
I'll remove the test in a followup commit. 👍🏻